ios/RNSScreen.h

```diff
@@ -69,6 +70,7 @@ typedef NS_ENUM(NSInteger, RNSActivityState) {
 @property (nonatomic) RNSScreenReplaceAnimation replaceAnimation;
 
 - (void)notifyFinishTransitioning;
+- (RCTTouchHandler *)reactTouchHandler;
```

Comment:
this should be `touchHandler` and not `reactTouchHandler` I believe

Reply:
Do we need it in `RNSScreen.h`?